plugins/modules/scc_product.py

```diff
@@ -16,6 +16,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import absolute_import, division, print_function
+from distutils.version import LooseVersion
```

Comment:
Oh, could you please move this import to line 60 where the other Ansible-related imports are? Thanks!